specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -150,10 +192,11 @@ export function IntegratedRecordSelector({
   relationship,
```

Comment:
Once this branch is based on xml-editor, could you please move this component to be above `RecordSelectorFromCollection`? That would make much more sense as the current order is wrong (i.e, naturally programmers read code from top to bottom, so components should be laid out in that order too) OR just move RecordSelectorFromCollection into a separate file